### DISCUSSION TITLE
Github large file error

### DISCUSSION THREAD

**Discussion by JulianHSC** [community/community#39089] [New to GitHub]
I hope this isn't a duplicate. A had a github large file error. I hope someone can help. My reply was in here but no reply at all: https://github.com/orgs/community/discussions/36651.

git -c diff.mnemonicprefix=false -c core.quotepath=false --no-optional-locks push -v --tags --set-upstream origin master:master
Pushing to https://github.com/xxx/Mobile-Touch-Input
Uploading LFS objects: 100% (294/294), 11 MB | 0 B/s, done.
POST git-receive-pack (chunked)
remote: warning: File Library/PackageCache/com.unity.burst@1.6.6/.Runtime/libburst-llvm-10.dylib is 83.53 MB; this is larger than GitHub's recommended maximum file size of 50.00 MB
remote: warning: File Library/PackageCache/com.unity.burst@1.6.6/.Runtime/libburst-llvm-11.dylib is 87.83 MB; this is larger than GitHub's recommended maximum file size of 50.00 MB
remote: warning: File Library/PackageCache/com.unity.burst@1.6.6/.Runtime/hostlin/lld is 57.95 MB; this is larger than GitHub's recommended maximum file size of 50.00 MB
remote: warning: File Library/PackageCache/com.unity.burst@1.6.6/.Runtime/hostmac/dsymutil is 64.13 MB; this is larger than GitHub's recommended maximum file size of 50.00 MB
remote: error: Trace: 8d039bb02e9546ef9a1a2e924629c03c831496dc6484f4f876f9b97824dc38b3

**Comment by DSDmark** [comment]
Hi @JulianHSC,

I'm not sure, but could try These commend according to GitHub documentation:-
https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github#removing-files-from-a-repositorys-history

And I think there is same type of issue, did you try that:-
https://stackoverflow.com/questions/37532228/how-do-i-fix-this-error-with-git

**Comment by david-engelmann** [comment]
[You may need to configure Github Large File Storage, this is a great guide on how to do so!](https://docs.github.com/en/repositories/working-with-files/managing-large-files/configuring-git-large-file-storage)
